src/utils/getRandomNumber/index.test.js

```diff
@@ -0,0 +1,9 @@
+import { getRandomNumber } from "./index";
+
+describe("getRandomNumber function", () => {
+  test("generates a 6-digit number", () => {
+    const digit = 6;
+    const randomNumber = getRandomNumber(digit);
+    expect(randomNumber.toString().length).toBe(6);
```

Comment:
we should here use `digit` instead of `6` directly.

Reply:
done